lib/Wrapper/OptimizingScheduler.cpp

```diff
@@ -40,7 +40,7 @@
 using namespace llvm::opt_sched;
 
 // hack to print spills
-bool OPTSCHED_gPrintSpills;
+extern bool OPTSCHED_gPrintSpills;
```

Comment:
Actually, this is the one which should be NOT extern. Rather, the declaration in the patch files (and our fork of llvm-project) should be marked extern.